README.md

```diff
@@ -1,10 +1,9 @@
 # Perform fast queries in R against a massive database of complete GWAS summary data
 
 <!-- badges: start -->
-[![Lifecycle:
-experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
-[![R build status](https://github.com/MRCIEU/ieugwasr/workflows/R-CMD-check/badge.svg)](https://github.com/MRCIEU/ieugwasr/actions)
+[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
 [![codecov](https://codecov.io/github/mrcieu/ieugwasr/branch/master/graphs/badge.svg)](https://codecov.io/github/mrcieu/ieugwasr) 
```

Comment:
It turns out all IEU package codecov URLs need to updated (I did the TwoSampleMR one the other day).

However this repo has additionally been deactivated on codecov - so you could delete for now (or ask Gib to reactivate on codecov - I don't think I have access to IEU codecov account).